model/system/metrics.yaml

```diff
@@ -1,4 +1,17 @@
 groups:
+  # system.* metrics
+  - id: metric.system.uptime
+    type: metric
+    metric_name: system.uptime
+    stability: experimental
+    brief: "The time the system has been running"
+    note: |
+      Instrumentations SHOULD use counter with type `double` and measure uptime with the highest precision available.
+      The actual accuracy would depend on the instrumentation and operating system.
+    instrument: counter
+    unit: "s"
+    attributes: []
```

Comment:
super-nit:

```suggestion
```

I know the schema requires it which results in the VSCode showing violation, but it's a bug and they are not required by anything.

Reply:
removed